language/stdlib/modules/LibraConfig.move

```diff
@@ -73,6 +73,33 @@ module LibraConfig {
         reconfigure_();
     }
 
+    // Set a config item to a new value and trigger a reconfiguration.
+    public fun set_with_capability<Config: copyable>(
+        _cap: &ModifyConfigCapability<Config>,
+        payload: Config
+    ) acquires LibraConfig, Configuration {
+        let addr = CoreAddresses::LIBRA_ROOT_ADDRESS();
+        assert(exists<LibraConfig<Config>>(addr), ECONFIG_DOES_NOT_EXIST);
+        let config = borrow_global_mut<LibraConfig<Config>>(addr);
+        config.payload = payload;
+        reconfigure_();
+    }
+
+    // Publish a new config item. The caller will use the returned ModifyConfigCapability to specify the access control
+    // policy for who can modify the config.
+    public fun publish_new_config_with_capability<Config: copyable>(
+        config_account: &signer,
+        payload: Config,
+    ): ModifyConfigCapability<Config> {
+        assert(Roles::has_on_chain_config_privilege(config_account), ENO_CONFIG_PRIVILEGE);
```

Comment:
Add `        assert(LibraTimestamp::is_genesis(), ENOT_GENESIS);`?

Reply:
Actually, question here: is it possible we'll want to publish more configs in the future, i.e. post-genesis?

Reply:
I think so, but the way we've be handling situations like this elsewhere is to use the most constrained configuration now (easier for verification) with the understanding that we can relax things later. As long as we don't need to modify a resource layout, it's fairly easy to make changes.

Reply:
I see, I was trying to unify the code of `publish_new_config` and `publish_new_config_with_capability`, do you think for both it makes sense to include the assert that we are calling from genesis? The `publish_new_config` is called for LibraVersion, RegisteredCurrencies and LibraVM, and `publish_new_config_with_capability` only called for LibraSystem.